static/js/user_status_ui.js

```diff
@@ -1,21 +1,14 @@
 "use strict";
 
 exports.input_field = function () {
-    return $(".user_status_overlay input.user_status");
+    return $(".update_status_text input.user_status");
 };
 
 exports.submit_button = function () {
-    return $(".user_status_overlay .set_user_status");
+    return $(".update_status_text .set_user_status");
 };
 
-exports.open_overlay = function () {
-    const overlay = $(".user_status_overlay");
-    overlays.open_overlay({
-        name: "user_status_overlay",
-        overlay,
-        on_close() {},
-    });
-
+exports.status = function () {
```

Comment:
`status` is not very descriptive. `edit_status` would be better.

Reply:
okay

Reply:
on second thought `edit_status_message` would be even better